### PR TITLE
Example of encoding a TCG DICE CMW

### PR DESCRIPTION
**Pull request by ounsworth**
This PR's not super clean because it's sitting on top of the changes for #42.

The relevant change is this:

https://github.com/lamps-wg/csr-attestation/pull/44/files#diff-9b88f7a24d6eff606780b418b2e2d8ff3e7a400513a2249abd64b87c26686bcf

### CODE REVIEW

**Review by nedmsmith — APPROVED** [review]
LGTM

File: CSR-ATTESTATION-2023.asn [hunk 1]
```diff
@@ -14,28 +14,31 @@ Certificate
      mechanisms(5) pkix(7) id-mod(0) id-mod-pkix1-explicit-02(51)}
 
 
-EXTENSION, ATTRIBUTE, AttributeSet{}, SingleAttribute{}, id-pkix 
+EXTENSION, ATTRIBUTE, AttributeSet{}, SingleAttribute{}
     FROM PKIX-CommonTypes-2009 -- from [RFC5912]
     { iso(1) identified-organization(3) dod(6) internet(1) security(5)
       mechanisms(5) pkix(7) id-mod(0) id-mod-pkixCommon-02(57) }
 
-id-aa
```
Comment (carl-wallace) [inline CSR-ATTESTATION-2023.asn:22]: You should put this back. The only reason I removed it was that I did not have SecureMimeMessageV3dot1 in the collection of ASN.1 modules I use to test compile. It is there though.

File: CSR-ATTESTATION-2023.asn [hunk 2]
```diff
@@ -14,28 +14,31 @@ Certificate
...
-FROM SecureMimeMessageV3dot1
-    { iso(1) member-body(2) us(840) rsadsi(113549)
-        pkcs(1) pkcs-9(9) smime(16) modules(0) msg-v3dot1(21) }
   ;
 
+id-aa OBJECT IDENTIFIER ::= 
+    { iso(1) member-body(2) us(840) rsadsi(113549)
+        pkcs(1) pkcs-9(9) smime(16) 9 }
+
+id-pkix OBJECT IDENTIFIER ::= {iso(1) identified-organization(3) dod(6) internet(1) security(5)
```
Comment (carl-wallace) [inline CSR-ATTESTATION-2023.asn:28]: This one should be imported from:

```
  id-pkix
  FROM PKIX1Explicit-2009
      {iso(1) identified-organization(3) dod(6) internet(1) security(5)
      mechanisms(5) pkix(7) id-mod(0) id-mod-pkix1-explicit-02(51)}
```

File: CSR-ATTESTATION-2023.asn [hunk 3]
```diff
@@ -14,28 +14,31 @@ Certificate
...
+     mechanisms(5)7 }
 
 -- Branch for attestation statement types
-id-ata OBJECT IDENTIFIER ::= { id-pkix ata(TBD1) }
```
Comment (carl-wallace) [inline CSR-ATTESTATION-2023.asn:30]: Leave the TBDs for the spec. It's common practice to use fake values locally for test compilation purposes. The TBDs are needed by IANA.

**Pull request merged**